DetoxSync/DetoxSync/SyncResources/DTXUISyncResource.m

```diff
@@ -113,45 +113,45 @@ - (void)_untrackForParam:(NSUInteger*)param eventIdentifier:(NSString*(NS_NOESCA
 - (void)trackViewNeedsLayout:(UIView *)view
 {
 	NSString* identifier = [self _trackForParam:&_viewNeedsLayoutCount eventDescription:_DTXStringReturningBlock(@"View Layout") objectDescription:_DTXStringReturningBlock(view.__detox_sync_safeDescription)];
-	
+
 	__detox_sync_orig_dispatch_async(dispatch_get_main_queue(), ^ {
-		[self _untrackForParam:&_viewNeedsLayoutCount eventIdentifier:_DTXStringReturningBlock(identifier)];
+		[self _untrackForParam:&self->_viewNeedsLayoutCount eventIdentifier:_DTXStringReturningBlock(identifier)];
```

Comment:
being explicit is indeed better

Reply:
but except for being more explicit, is there any other goal in this change? What exactly is the fix here?

Reply:
Ah, yes. Sorry. This fixes an XCode warning:
https://github.com/wix/DetoxSync/pull/33/files#diff-08fdd5ea3608159a1a2dcf8f2badd094a37e7752afd55967213368f32ae64c68L1227